api/api_listagem-clientes/api/routes/index.js

```diff
@@ -0,0 +1,8 @@
+const bodyParser = require('body-parser')
+const clientes = require('./clientesRoute')
+
+module.exports = app => {
+    app.use(bodyParser.json())
```

Comment:
Utilizar o parser do express.

```suggestion
    app.use(express.json())
```